htdocs/editicons.bml

```diff
@@ -117,32 +119,7 @@ use strict;
 
             my $MAX_UPLOAD = LJ::Userpic->max_allowed_bytes($u);
             my $error;
-
-            # act like the factory is disabled if no mogile
-            my $factory_disabled = !LJ::is_enabled('userpicfactory') || !LJ::mogclient();
```

Comment:
It looks like we don't allow the userpicfactory to be disabled any more? This removes a check for LJ::is_enabled('userpicfactory') from /editicons - not that it was ever documented in the config files anywhere that I can see.  Do you know why it was allowed to be disabled in the past?